FirebaseSessions/Sources/SessionGenerator.swift

```diff
@@ -19,13 +19,15 @@ import Foundation
 
 struct SessionInfo {
   let sessionId: String
-  let previousSessionId: String?
+  let firstSessionId: String
   let shouldDispatchEvents: Bool
+  let sessionIndex: Int32
```

Comment:
Can we use native types like `Integer`?

Reply:
In Swift, `Int32` is a native type - I think `Integer` is just an interface. Using `Int32` makes it compatible with our proto which also uses int32

https://stackoverflow.com/questions/24053432/int-vs-integer-in-swift

<img width="1285" alt="Screenshot 2023-02-01 at 1 52 32 PM" src="https://user-images.githubusercontent.com/555046/216136056-bcc259b7-1b0c-4e37-8e5d-b0ebeb798c28.png">